### ISSUE TITLE
How to use cyw43_arch?

### ISSUE THREAD

**Issue by Kernald** [dfr/rules_pico#11]
The Pico W board needs specific headers to control the wireless board, e.g. `pico/cyw43_arch.h`. These don't seem to be exposed by these rules yet, is that correct?

**Comment by dfr** [comment]
That is correct - when I wrote the rules, Pico W didn't exist. I don't think it would be too hard to add this support but I only have the original pico hardware so I have no way to test.

**Comment by Kernald** [comment]
I'll might look into that eventually - happy to test if you put a PR up before that :-)
